packages/client/tests/e2e/nextjs-schema-not-found/10_monorepo-serverComponents-customOutput-noReExport/_steps.ts

```diff
@@ -0,0 +1,28 @@
+import { $, cd, ProcessPromise } from 'zx'
+
+import { executeSteps } from '../../_utils/executeSteps'
+
+let nextJsProcess: ProcessPromise
+void executeSteps({
+  setup: async () => {
+    await $`pnpm install`
+    cd('packages/service')
+    await $`pnpm exec prisma db push --force-reset`
+    await $`pnpm exec next build`
+  },
+  test: async () => {
+    await $`rm -fr .next/standalone/node_modules/next`
+    nextJsProcess = $`node .next/standalone/server.js`
+    await $`sleep 5`
```

Comment:
Will it be possible to something more reliable? Like, for example, checking that the `3000` ports response to requests?

Reply:
Yes I think that is a good idea, it will also shorten the delay most likely. I think something like is is possible with `curl`.